.travis.yml

```diff
@@ -4,8 +4,12 @@ go:
   - 1.9
   - '1.10'
   - '1.11'
+  - '1.12'
   - master
 
+env:
```

Comment:
Why do we have still those `go get` in the Travis file then?

Reply:
Whether or not gomod is installed using go get

Reply:
I don't understand. Why do we have these?
```
install:
  - go get github.com/mattn/goveralls
  - go get github.com/stretchr/testify/assert
  - go get github.com/gorilla/websocket
  - go get github.com/onsi/ginkgo
  - go get github.com/onsi/gomega
  - go get golang.org/x/sync/semaphore
```
If we use go mod, what's the point of keeping manual go get like this?